src/tests/unit/events/event_post_1.f90

```diff
@@ -0,0 +1,20 @@
+program event_post_1
+  use iso_fortran_env
+  implicit none
```

Comment:
Sure, but since we have `event_post` 1 and 2 now, presumably they are testing different aspects of event post... I'll look through the code and see if I can come up with anything that distinguishes between them in a concise and meaningful way, otherwise I'll probably just leave it for now.